test-runner-frontend/src/Views/ReportUtils.js

```diff
@@ -0,0 +1,86 @@
+import html2canvas from 'html2canvas';
+import jsPdf from 'jspdf';
+
+export const builChartsData = (from) => {
+  const classesAndBenchmarks = {};
+  from?.statistics?.map((testRun, index) => {
+    var benchmarkCassAndMethod = testRun.benchmark.split('.');
```

Comment:
can we add a comment above this line with an example of a benchmark string? this will help whoever is look at this code to understand the different parts that are expected to be split out

Reply:
Done